example/src/app/__snapshots__/app.component.spec.ts.snap

```diff
@@ -7,20 +7,20 @@ exports[`AppComponent snaps 1`] = `
   variableWithPrecedingDolar={[Function Number]}
 >
   <h1
-    class="ng-tns-c1-0"
+    class=""
   >
```

Comment:
Can you add a class value where a in-template-defined class remains in the snapshot? Maybe also some edge case like `custom-class25-0`.
A valid use case I can come up with right now is using classes for special padding or margin values like `margin-rem3-0`.